projectile.el

```diff
@@ -716,7 +716,7 @@ Files are returned as relative paths to the project root."
     ;; cache disabled or cache miss
     (or files-list
         (if (eq projectile-indexing-method 'native)
-            (projectile-dir-files-native root directory)
+            (projectile-remove-ignored (projectile-dir-files-native root directory))
```

Comment:
`projectile-dir-files-native` is doing the ignoring internally. If this is not happening than there's a bug in it.